### ISSUE TITLE
Не работает не на Хроме, не на Яндекс браузере.

### ISSUE THREAD

**Issue by AbrahamChutilian** [anatolykopyl/vk-adult#1]
Я попробовал на обеих браузерах и ни где не работает как скомпилировонная версия(1.2.1), так и версия с "Extension store"-a гугла.
Вот несколько проблем с которыми столкнулся:
1: `chrome.debugger is undefined`. Хотя в manifest.json-e он есть в permissions
2:` const filterWrap=document.querySelector(".video_search_filters_wrap")` тоже undefined потому, что елемент с таким названием класса нет. Я сумел добавить опцию, когда изменил эту часть вот так в Яндекс браузере:
```
	const filtersWraps = document.querySelectorAll(
		".vkuiFormItem.vkuiFormItem--withPadding.vkuiInternalFormItem.vkuiFormItem--sizeY-compact.vkuiInternalFormItem--sizeY-compact.vkuiFormItem--withTop.vkuiInternalFormItem--withTop"
	);

	const filtersWrap = filtersWraps[filtersWraps.length - 1];
```

**Comment by anatolykopyl** [comment]
В первую очередь, спасибо за репорт!
А во-вторых, можно скриншот страницы, где сработал `querySelector` с измененным значением?
Расширение работает только в интерфейсе vk.com/video, может быть оно на другой странице не сработало просто?

**Comment by AbrahamChutilian** [comment]
Да через vk.com/video работает, но на хроме. В яндексе, хоть и галочка появляется и птичка ставится, но все равно нечего не дает. 
В данном случае расшерение не работает, если пользаватся как обично я делаю, то есть сначала через скажем мою страницу вбивать название видео в вверху в строке поиска и только потом справа ставить галочку "Без ограничений". В таком случае и получается проблема с названиями класов.
Вот скриншоты с измененным `querySelector`.
Яндекс браузер
![vk screen](https://github.com/user-attachments/assets/13153f78-7578-4e71-9f04-d47f76a96678)
Хром
![vk-chrome](https://github.com/user-attachments/assets/6ea5b262-08b4-4bde-b5c2-b085a1c4da3c)
Как видно в разных браузерах DOM немного по другому собран и название классов тоже разные.

**Comment by anatolykopyl** [comment]
Да, действительно стоит сделать чтобы работало и в поиске на `/search`.
